### ISSUE TITLE
The app crashed! Nothing will work now; you have to refresh the page.

### ISSUE THREAD

**Issue by louwers** [a-b-street/abstreet#1005]
```
panicked at 'called `Result::unwrap()` on an `Err` value: Line from Pt2D { x: 5857.4217, y: 4813.787 } to Pt2D { x: 5857.4205, y: 4813.7771 } too small', geom/src/line.rs:25:29
```

To reproduce go to the following URL: https://play.abstreet.org/0.3.34/osm_viewer.html?system/nl/groningen/maps/huge.bin&--cam=19.31/53.21590/6.56499

**Comment by dabreegster** [comment]
With `RUST_BACKTRACE=1` locally, more info: ```
   3: geom::line::Line::must_new
   4: geom::polyline::PolyLine::last_line
   5: map_model::objects::lane::Lane::end_line
   6: map_gui::render::intersection::calculate_corners
   7: <map_gui::render::intersection::DrawIntersection as map_gui::render::Renderable>::draw
```

a polyline should get checked so that this is impossible, so let's see what's happening...

**Comment by dabreegster** [comment]
OK, this was fixed "properly" by #1063, will be in the next release!
